bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/internal/ImageList.java

```diff
@@ -445,4 +457,22 @@ public int size () {
 	return result;
 }
 
+public static long win32_getHandle(ImageList imageList, int zoom) {
+	if(!imageList.zoomToHandle.containsKey(zoom)) {
+		int scaledWidth = DPIUtil.scaleUp(DPIUtil.scaleDown( imageList.width,imageList.zoom), zoom);
+		int scaledHeight = DPIUtil.scaleUp(DPIUtil.scaleDown( imageList.height,imageList.zoom), zoom);
+		long handle = OS.ImageList_Create (scaledWidth, scaledHeight, OS.ILC_MASK | OS.ILC_COLOR32, 16, 16);
```

Comment:
Shouldnt't this use the flags of the original list?
```suggestion
		long handle = OS.ImageList_Create (scaledWidth, scaledHeight, imageList.flags, 16, 16);
```

Reply:
Good point. We store the flags in the image list now and use that for the creation of the scaled variant